integration/testdata/ca_certificate_apps/build/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+func main() {
+
+	// we think this will automatically	include the CA
+	// installed by the buildpack
```

Comment:
Have we confirmed that this is the case or should this comment still remain?

Reply:
We've confirmed this. 
```suggestion
```